nym-vpn-core/crates/nym-vpn-lib/src/platform/mod.rs

```diff
@@ -384,6 +386,16 @@ async fn start_vpn_inner(config: VPNConfig) -> Result<(), VpnError> {
     .await
 }
 
+fn log_build_info() {
+    let build_info = nym_bin_common::bin_info_local_vergen!();
+    tracing::info!(
+        "{} {} ({})",
+        build_info.binary_name,
+        build_info.build_version,
+        build_info.commit_sha
```

Comment:
How is this going to work for idempotent builds?

Reply:
If the build is idempotent, the commit_sha string will be something constant